### PR TITLE
UITAG-40 Update permission name for Tags settings

### PR DESCRIPTION
**Pull request by AndreiShumski2**
Update permission name for Tags settings
https://issues.folio.org/browse/UITAG-40

### CODE REVIEW

**Review by uladislausamets — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
         "displayName": "UI: Tags module is enabled"
       },
       {
-        "permissionName": "settings.tags.enabled",
```
Comment (aliaksei-chumakou) [inline package.json:30]: you can't delete such permissions, do you know any module where it's deleted? you can check stripes docs, they are required

Comment (AndreiShumski2) [reply package.json:30]: thanks. I was looking at `module.tags.enabled`

Comment (aliaksei-chumakou) [reply package.json:30]: from my perspective this PR is still wrong. It looks like yo need new permission, created according to the guidelines with translation,  based on "settings.tags.enabled" permission. 
at the same time "settings.tags.enabled" should become visible: false

Comment (AndreiShumski2) [reply package.json:30]: fixed

File: package.json [hunk 2]
```diff
@@ -28,10 +28,18 @@
       },
       {
         "permissionName": "settings.tags.enabled",
-        "displayName": "Settings (Tags): display list of settings pages",
+        "displayName": "Settings (Tags): Can enable or disable tags for all apps",
```
Comment (aliaksei-chumakou) [inline package.json:31]: I'm not sure that this name should be changed

**Review by aliaksei-chumakou — APPROVED** [review]

**Review by axelhunn — APPROVED** [review]

**Pull request merged**